BUILD.bazel

```diff
@@ -74,8 +75,31 @@ config_setting(
     values = {"compilation_mode": "opt"},
 )
 
-# === Begin of rpc definitions ===
+bool_flag(
+    name = "jemalloc_flag",
+    build_setting_default = True,
+)
+
+config_setting(
+    name = "jemalloc",
+    constraint_values = [
+        "@platforms//os:linux",
```

Comment:
Does it mean if this constraints are not satisfied, it won't be built with jemalloc?

Reply:
Yes. Right now I only enable it for linux on x86.